### PR TITLE
Static fixes

### PR DESCRIPTION
**Pull request by peterfpeterson**
Fixing the builds

**To test:**

Builds should pass.

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/Calibration/tube.py [hunk 1]
```diff
@@ -4,7 +4,8 @@
 Definition of Calibration
 =========================
 
-.. autofunction:: calibrate(ws, tubeSet, knownPositions, funcForm, [fitPar, margin, rangeList, calibTable, plotTube, excludeShorTubes, overridePeaks, fitPolyn, outputPeak])
```
Comment (martyngigg) [inline scripts/Calibration/tube.py]: Unfortunately this breaks one of things I was attempting to fix. `autofunction` doesn't seem to like the signature split across multiple lines

Comment (peterfpeterson) [inline scripts/Calibration/tube.py]: I welcome whatever fix you have. Just change it on the branch and push.

Comment (martyngigg) [inline scripts/Calibration/tube.py]: Can we put i back on 1 line & `#noqa` this?

**Comment by martyngigg** [comment]
`clang`, `flake8` & doctests are good. :shipit:

**Pull request merged**